.github/workflows/publish.yml

```
@@ -0,0 +1,153 @@
+          ref: ${{ github.event.release.tag_name }}
+
+      - name: Check if version already exists on PyPI
+        id: pypi_check
+        env:
+          RELEASE_TAG: ${{ github.event.release.tag_name }}
+          PACKAGE_NAME: azure-pipeline-validator
+        run: |
+          set -euo pipefail
+          version="${RELEASE_TAG#v}"
+          skip_publish="$(python - <<'PY'
+import json
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)